Resources

joetsoi opened the pull request:
@mintcanary, @morchickit This is what this does so far.
![open_data_resources_-_2015-04-20_18 18 28](https://cloud.githubusercontent.com/assets/241130/7235983/b71d179e-e789-11e4-97c0-e33a104df9eb.png)

The source data needs to be cleaned up a bit it seems, there is one column with a huge list of countries, additionally there are some columns with whitespace 'Open Data ' vs 'Open Data'

Once there are the other place holders for the resource pages I can link it all together.

please don't merge this yet!

joetsoi committed:
change resource fetcher

* fetch google doc as csv to avoid login requirement
* output as yaml files in _data directory

joetsoi committed:
Merge branch 'gh-pages' into resources

joetsoi committed:
resources side menu

* resources now all in single yaml file
* yaml file generation script moved to /scripts
* side menu liquid template generates count

morchickit commented:
Hi @joetsoi , it might be easier if you'll drop the country indetifier and leave only region. Can you do that?

morchickit commented:
Also, I'll clean the list so it will fit our new themes.

joetsoi commented:
yeah removing the country is easy, the list of columns that will appear is defined in https://github.com/okfn/opendatahandbook-v2/blob/resources/_data/resources_filters.yaml so I can just remove it from there.

morchickit commented:
Great, let's remove it then. :-)
I'll sort the topics.

joetsoi committed:
[open data resources]

make resources a collection
move script to /scripts
javascript filter for side menu

joetsoi committed:
change Country to Region

joetsoi committed:
[open data resources] hide search placeholder for now

joetsoi committed:
[open data handbook] add slugify to requirements

joetsoi committed:
Merge branch 'gh-pages' into resources

Conflicts:
	_config.yml
	_includes/resources-menu.html

joetsoi committed:
[open data resources] add MediaType and fix links

joetsoi committed:
[open data resources] fix class of topic

joetsoi commented:
@mintcanary , could you test that this works before merging? the filtering from the side menu should work, I've just hidden the search box for now.

I haven't hooked up the javascript that does the filtering to the Featured Topics and media type links on the landing, but that shouldn't stop this being merged first.

I've broken the css of the "Open Website" links somehow, you might be quicker at fixing this
![open_data_resources_-_2015-04-27_11 30 18](https://cloud.githubusercontent.com/assets/241130/7345866/e2d96c24-ecd0-11e4-8334-b799f54057d9.png)
you can see the links are no longer a green button

smth commented:
@joetsoi ~~When I try this locally, the filter list and results list are both empty..~~ <- Ignore that, I think I just needed to reload the config.

Looking good. The button styling was broken by an unclosed `p` https://github.com/okfn/opendatahandbook-v2/pull/38/files#diff-5ea558b64bbbabde6194524658e252a2R29

joetsoi committed:
[open data resources] correct type in Language

joetsoi commented:
odd that should only occur if it hasn't found the resources collection, what's the jekyll version you're using.

smth commented:
@joetsoi Sorry, you probably didn't see my edit (I just needed to reload the config).

joetsoi commented:
cool, I'll fix the styling and merge tthis in then?

joetsoi committed:
[open data resources] fix p closing tag

smth commented:
Yes please.

joetsoi commented on js/resources.js:
```diff
@@ -0,0 +1,19 @@
+$(document).ready(function() {
+    $("#menu .filter .filter-items > a").click(function() { 
+        $("#menu .filter .filter-items > a").removeClass("active");  
+        $(this).toggleClass("active");
```
should fix the removeClass("active") so it doesn't remove it from the current selected. This way the toggle in the next line would let you deselect the current filter.
